packages/filebrowser/src/listing.ts

```diff
@@ -2114,6 +2272,21 @@ export namespace DirListing {
       return DOMUtils.findElement(node, ITEM_TEXT_CLASS);
     }
 
+    /**
+     * Get the checkbox input element node.
+     *
+     * @param node A node created by [[createItemNode]] or
+     * [[createHeaderItemNode]]
+     *
+     * @returns The checkbox node.
+     */
+    getCheckboxNode(node: HTMLElement): HTMLInputElement | null {
+      const checkbox = node.querySelector(
+        `.${CHECKBOX_WRAPPER_CLASS} input[type=checkbox]`
+      ) as HTMLInputElement | void;
+      return checkbox || null;
```

Comment:
Minor styling we could use for consistency:

```suggestion
      const checkbox = node.querySelector(
        `.${CHECKBOX_WRAPPER_CLASS} input[type=checkbox]`
      ) as HTMLInputElement | null;
      return checkbox ?? null;
```

Reply:
Oh right! querySelector's return signature is actually [Element | null](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector).

I fixed it, thanks for pointing it out